Plugins/NativeEngine/Source/ShaderCompilerTraversers.cpp

```diff
@@ -5,12 +5,15 @@
 #include <glslang/MachineIndependent/RemoveTree.h>
 
 #include <bgfx/bgfx.h>
+#include <bx/bx.h>
```

Comment:
is this necessary?

Reply:
Currently using it for their static assert and array count implementation (L476), This may not be necessary, however.

Reply:
Ok, it's fine so that you can copy and paste. Is it a copy and paste?

Reply:
yes, it is a copy paste